index.html

```diff
@@ -1,168 +1,309 @@
+          <h3>Github</h3>
+          <p>Browse, branch, fork or clone from Github.</p>
+          <a class="external box__link" href="https://github.com/openstack/cloud-init">Github repository</a>
+        </div>
+      </div>
 
-    <a href="https://bugs.launchpad.net/cloud-init/+filebug">Report a bug</a>
-    <a href="http://pingx.net/freenode/cloud-init">Chat on freenode</a>
-    <a href="http://cloudinit.readthedocs.org/">Read the docs</a>
-  </div>
-</section>
+      <div class="four-col last-col">
+        <div class="box">
+          <img src="https://assets.ubuntu.com/v1/4cd0df1c-picto-download-orange.svg" alt="Download logo">
+          <h3>Download</h3>
+          <p>Download the offically released archives.</p>
+          <a class="external box__link" href="https://launchpad.net/cloud-init/+download">Download project files</a>
+        </div>
+      </div>
+    </div>
+  </section>
 
-<!-- About / The Team -->
-<section class="row">
+  <!-- Public Cloud Examples -->
+  <section class="row">
+    <div class="twelve-col">
+      <h2>Used across the public cloud</h2>
+      <p>Amazon, Google, Azure, Openstack, MAAS, and a long list of Ubuntu
+        Certified Public Cloud partners support cloud-init.
+      </p>
 
-  <div class="six-col">
+      <ul class="inline-logos">
+        <li class="inline-logos__item">
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/a3729935-logo-rackspace.png" alt="Rackspace logo">
+        </li>
+        <li class="inline-logos__item">
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/39e011de-logo-ovh-300DPI.png" alt="OVH logo">
+        </li>
+        <li class="inline-logos__item">
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/788696ad-logo-vmware.png" alt="WMware logo">
+        </li>
+        <li class="inline-logos__item">
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/2b5837e9-logo-microsoft-azure.svg" alt="Microsoft Azure logo">
+        </li>
+        <li class="inline-logos__item">
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/25b3d138-201509-amazon_aws_4c_en.svg" alt="Amazon Web Services logo">
+        </li>
+        <li class="inline-logos__item">
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/f7f8709c-logo-joyent.svg" alt="Joyent logo">
+        </li>
+        <li class="inline-logos__item">
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/4f54c31c-Google-Cloud-Platform.png" alt="Google Cloud logo">
+        </li>
+        <li class="inline-logos__item">
+            <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/9aadcd6b-partner-logo-fujitsu.png" alt="Fujitsu logo">
+        </li>
+      </ul>
+      <a href="#_" class="button--primary right">See our other Ubuntu public cloud partners</a>
```

Comment:
@anthonydillon I don't have a link to put here - have requested it from Frankie.